pkg/core/blockchain.go

```diff
@@ -1038,49 +1037,35 @@ func (bc *Blockchain) handleNotification(note *state.NotificationEvent, d dao.DA
 	if !ok || len(arr) != 3 {
 		return
 	}
-	var from []byte
-	fromValue := arr[0].Value()
-	// we don't have `from` set when we are minting tokens
-	if fromValue != nil {
-		from, ok = fromValue.([]byte)
-		if !ok {
-			return
-		}
+	from, err := parseUint160(arr[0])
+	if err != nil {
+		return
 	}
-	var to []byte
-	toValue := arr[1].Value()
-	// we don't have `to` set when we are burning tokens
-	if toValue != nil {
-		to, ok = toValue.([]byte)
-		if !ok {
-			return
-		}
```

Comment:
Initial intention was to explicitly prohibit all other item types (loose type check). But I agree the amount of such events is negligible.

Reply:
Lacking neo-project/neo#1879 we really can get just about anything here. Also:
https://github.com/neo-project/neo-modules/blob/ef59a0eb27fb23c49610c0362e1fe7d9d0ec7839/src/RpcNep17Tracker/RpcNep17Tracker.cs#L135-L138
IIRC `GetSpan()` will do the same and extract uint160 from integer if it's there.